.travis.yml

```diff
@@ -22,27 +22,32 @@ jobs:
   include:
     # Tests, only when not triggered by the daily cron.
     - stage: static
+      name: static tests
```

Comment:
*gasp* what is this madness?

Reply:
it is, it evens export the name into the environment to do neat things